### ISSUE TITLE
BLE notification

### ISSUE THREAD

**Issue by delaram-mz** [thunkable/thunkable-issues#651]
I am using Thunkable X and I need to get notified every time a variable in BLE stack changes.
so please add a feature so this could be possible.

**Comment by josmas** [comment]
This is not something we will be working on any time soon, but we will consider it in the future. Thanks for your request.

**Comment by AnatiliyKsl** [comment]
This is somehow not logical!There is an application, there are clients who want to use it, do you have constant questions about BLE?  Team Thunkable is simply not responding to this problem! Strange! WIFI, BLE IS THE ONLY OPPORTUNITY TO CONTROL EXTERNAL DEVICES! Your application becomes uninteresting for most people interested in controlling with a smartphone.

**Comment by junliaw** [comment]
For our maker class,it's a very useful feature. Hope you can implement it soon.thank you.

**Comment by fender21** [comment]
Here to chime in on adding a Notify mechanism. I am having to evaluate other platforms right due to this Thunkable limitation which is a shame as I am a big fan of Thunkable.

**Comment by marcelocsi** [comment]
> Here to chime in on adding a Notify mechanism. I am having to evaluate other platforms right due to this Thunkable limitation which is a shame as I am a big fan of Thunkable.

I totally agree, the previous version of Thunkable had notify feature in there, also MIT Invertor does support the notify feature as well.

**Comment by jmrmaker** [comment]
I would also like to request a notify feature and without it, I will have to give up on Thunkable like so many others.  It's a shame that this keeps getting rejected with the number of requests I've come across.
